src/term/readback/non_linear.rs

```diff
@@ -60,73 +59,188 @@ pub fn net_to_term(
 }
 
 // BTreeSet for consistent readback of dups
```

Comment:
Update comment here

Reply:
You can just remove it since it's implied by the type. (the only reason we ever use IndexMap is for insertion ordered iteration)